src/components/CollapsibleFeedback/Feedback.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Collapsible, Icon } from '@edx/paragon';
+import { ExpandMore, ExpandLess } from '@edx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import messages from './messages';
+import InfoPopover from 'components/InfoPopover';
+
+const Feedback = ({
+  criterionName,
+  selectedOption,
+  selectedPoints,
+  commentHeader,
+  commentBody,
+  defaultOpen,
+}) => {
+  const [isExpanded, setIsExpanded] = React.useState(defaultOpen);
+  const { formatMessage } = useIntl();
+
+  return (
+    <>
+      <div className='mt-2'>
+        <div className='d-flex justify-content-between align-items-center'>
+          <h5 className='mb-0'>{criterionName}</h5>
+          <InfoPopover onClick={() => {}}>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```

Comment:
uuuuugh.  we need to get these messages still

Reply:
Actually, these messages should come from the ora config data, as in the ESG rubric